nautobot_firewall_models/tests/test_api.py

```diff
@@ -33,10 +33,11 @@ def setUpTestData(cls):
     def test_unique_validators(self):
         """Test the unique validators for IPRange."""
         # Add object-level permission
-        obj_perm = ObjectPermission(name="Test permission", actions=["add", "change"])
+        obj_perm = ObjectPermission(name="Test permission", actions=["add", "change", "view"])
         obj_perm.save()
         obj_perm.users.add(self.user)
         obj_perm.object_types.add(ContentType.objects.get_for_model(self.model))
+        obj_perm.object_types.add(ContentType.objects.get_for_model(VRF))
```

Comment:
Fix to account for core now enforcing permissions on related objects in the API. https://github.com/nautobot/nautobot/pull/7031